device_attributes/index.bs

```diff
@@ -0,0 +1,192 @@
+<pre class='metadata'>
+Title: Device Attributes API
+Shortname: device-attributes
+Level: 1
+Editor: Anqing Zhao 130190, Google LLC https://www.google.com, anqing@google.com
+Status: w3c/CG-DRAFT
+URL: https://wicg.github.io/WebApiDevice/device_attributes
+Abstract: This document defines a web platform API that enables developers to query device information (device ID, serial number, location, etc) from managed devices. This API is invaluable for such use cases as Virtual Desktop Infrastructure (VDI) and Context-based Configuration.
+Repository: https://github.com/WICG/WebApiDevice
+Group: wicg
+</pre>
+ 
+<pre class=link-defaults>
+spec:html; type:interface; text:Navigator
+spec:url; type:dfn; text:origin
+spec:html; type:dfn; for:/; text:global object
+spec:webidl; type:dfn; text:resolve
+spec:webidl; type:dfn; text:record
+</pre>
+ 
+<pre class="anchors">
+ 
+</pre>
+ 
+# Model # {#model}
+ 
+## Managed devices ## {#managed-devices}
+ 
+The API is presumed to be used on devices which are not fully controlled by the end user, but rather by an external entity, the device administrator. Device administrators are given the power to fully control the managed devices.
+ 
+## Managed web applications ## {#managed-web-applications}
+ 
+The API is presumed to be used by web applications, which are understood and managed by the device administrators.
+ 
+## Device attributes ## {#device-attributes}
+ 
+The API is presumed to be used with a management infrastructure which allows administrators to manage a fleet of devices and configure properties for those devices.
```

Comment:
This specification could use a non-normative section with examples of how these properties are set up in a couple different ecosystems. Including Chrome's, Edge's, and Safari's enterprise management systems would help give assurance that it's possible to implement this interoperably.

Reply:
Hmm, I am not very familiar with Edge's Safari's enterprise management systems. Our current goal is to support these device attributes API in Chrome OS only. Please allow me to confirm with you whether this is a mandatory requirement for publishing this draft specification.

Reply:
It's definitely not a requirement before we publish a specification, and publishing the specification is a good way to attract some attention from folks who are familiar with other enterprise management systems.

It is a [requirement](https://www.w3.org/2020/Process-20200915/#implementation-experience) that the spec can be implemented on other enterprise management systems before the W3C will publish a [Candidate Recommendation](https://www.w3.org/2020/Process-20200915/#transition-cr). However, Chrome often ships things before getting to CR: that decision will be up to the API owners based on https://www.chromium.org/blink/guidelines/web-platform-changes-guidelines. When you submit this PR, it'll be a good idea to try to reach out to those other groups, to validate that the shape is right.

Reply:
Thanks for the detailed introduction and those links. I'd like to follow your suggestion that let's submit this PR first, then collect more ideas from others.

I have already solved (or properly handled) all your other comments. Please let me know whether I can get your approval. Thanks for all your help!